cards.html

```diff
@@ -0,0 +1,342 @@
+                </a>
+            </li>
+
+            <li>
+                <a href="#" class="navbar-list-item icons">
+                    <i class="fas fa-moon"></i>
+                </a>
+            </li>
+
+        </ol>
+    </nav>
+    <section class="doc">
+
+        <!-- sidebar -->
+        <section class="component-navbar ">
+            <div class="sidebar">
+                <h1 class="component-navbar-heading">Getting Started</h1>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">usage</a>
+                </li>
+                <div class="line-hori side mg-16"></div>
+                
+                <h1 class="component-navbar-heading">Components</h1>
+                <ul>
+                    <li class="component-navbar-item">
+                        <a href="avatar.html" class="component-navbar-item-link">Avatar</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="badge.html" class="component-navbar-item-link">badge</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="alert.html" class="component-navbar-item-link">Alert</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="button.html" class="component-navbar-item-link">button</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="#" class="component-navbar-item-link">card</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="images.html" class="component-navbar-item-link">images</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="#" class="component-navbar-item-link">Alert</a>
+                    </li>
+                    <li class="component-navbar-item">
+                        <a href="#" class="component-navbar-item-link">Alert</a>
+                    </li>
+                    
+                    
+                
+                </ul>
+            </div>
+        </section>
+
+        <!-- centre documentation  -->
+        <section class="component-doc">
+            <h1 class="component-doc-no">05</h1>
+
+            <!-- card with badge-->
+            <h1 class="component-navbar-heading">card with badge</h1>
+            <h3 class="component-navbar-item">In computing, an avatar
+                 (also known as a profile picture or userpic) 
+                 is a graphical representation of a user or the
+                  user's character or persona.</h3>
+ 
+            <div class="component-avatar flex flex-nowrap flex-centre">
+                <div class="card-basic">
+                <span class="badge badge-warning postion">hot</span>
```

Comment:
Here's a typo error. Idk why you did this but if possible correct it and also you have created flex utilities but you r repeating those in your cards. you could have used the classes that you defined for it.